app/helpers/patient_query_helper.rb

```diff
@@ -492,11 +492,14 @@ def advanced_filter_relative_date(patients, field, filter, tz_diff, type)
 
       case filter[:value][:operator]
       when 'less-than'
+        # subtract one day to the timespan if relative date field does not have have a timestamp
```

Comment:
can we include an example of this in the comments for dates with/without timestamps to better illustrate what's going on here

Reply:
added here: https://github.com/SaraAlert/SaraAlert/pull/765/commits/4da3879b49d073d7b228342aa69de01be06e9032